deploy/infra/bin/common-fate.ts

```diff
@@ -40,6 +40,16 @@ const analyticsDeploymentStage = app.node.tryGetContext(
   "analyticsDeploymentStage"
 );
 
+let shouldRunCronHealthCheckCacheSync = app.node.tryGetContext(
+  "enableCronHealthCheck"
+);
+
+if (typeof shouldRunCronHealthCheckCacheSync === "string") {
```

Comment:
I believe in typescript/js you can use something like
```
someString = "true"
enabled = !!someString
```

which will cast the string to boolean

Reply:
<img width="115" alt="image" src="https://user-images.githubusercontent.com/21688404/219561503-110bb439-aa8a-405c-9345-4499e442e77b.png">
🤧

Reply:
Not quite the neatest formatting but I think it covers the check, it also will covers `True` -> `true`. Happy to alter if you have any other suggestions?